### PR TITLE
add "specialScriptTags"

### PR DESCRIPTION
**Pull request by hryvinskyi**
This change is [Reviewable](https://reviewable.io/reviews/voku/simple_html_dom/55)

### CODE REVIEW

**Comment by voku** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- src/voku/helper/HtmlDomParser.php  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/voku/simple_html_dom/pullRequest?prid=5956217&bid=19521050)

File: src/voku/helper/HtmlDomParser.php [hunk 1]
```diff
@@ -55,6 +55,26 @@ class HtmlDomParser extends AbstractDomParser
...
+     *     'text/html' => 'text\/html',
+     *     'text/x-custom-template' => 'text\/x-custom-template',
+     *     'text/x-handlebars-template' => 'text\/x-handlebars-template'
+     * ]
+     * ```
+     *
+     * @var string[]
+     */
+    protected $specialScriptTags = [
+        'text/html' => 'text\/html',
```
Comment (voku) [inline src/voku/helper/HtmlDomParser.php:73]: Maybe we can use "preg_quote()" + instead of this duplicate key⇒value pair? What do you think?

Comment (hryvinskyi) [reply src/voku/helper/HtmlDomParser.php:73]: Yes, I think so its better to use preg_quote. Thanks.

**Comment by voku** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- src/voku/helper/HtmlDomParser.php  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/voku/simple_html_dom/pullRequest?prid=5956217&bid=19521050)

File: src/voku/helper/HtmlDomParser.php (hunk 1 above)
Comment (voku) [reply src/voku/helper/HtmlDomParser.php]: Thanks for your contribution. :+1:

**Pull request merged**